src/modules/Transcript/PriceChart/index.tsx

```diff
@@ -301,16 +246,117 @@ export function PriceChartUI(props: PriceChartUIProps): ReactElement {
 
 /** @notExported */
 export interface PriceChartProps extends PriceChartSharedProps {
+    eventId: string;
     headerExpanded: boolean;
 }
 
+const getPriceQuery = gql`
```

Comment:
Mind renaming to `GetPriceQuery` or just `PriceQuery`? Silly, but for some reason I keep expecting this to be a function, not a constant.

Reply:
👍